x-pack/plugins/remote_clusters/__jest__/client_integration/list/remote_clusters_list.test.js

```diff
@@ -50,14 +39,13 @@ describe('<RemoteClusterList />', () => {
   });
 
   describe('when there are no remote clusters', () => {
+    let waitFor;
     let exists;
-    let component;
 
     beforeEach(async () => {
-      ({ exists, component } = setup());
+      ({ waitFor, exists } = setup());
```

Comment:
We determined that it is not best practice to use the `waitFor` helper in tests.

Reply:
Yep, no more `waitFor` in our jest tests 🙏